base/uk.ac.stfc.isis.ibex.scriptgenerator/src/uk/ac/stfc/isis/ibex/scriptgenerator/generation/GeneratedLanguage.java

```diff
@@ -16,6 +16,10 @@
 	
 	private final String name;
 	
+	/**
+	 * Constructor
```

Comment:
```suggestion
	 * Constructor.
```

Also, can we make this say something more useful? Currently this will fix checkstyle but actively make our code worse by adding noise with no useful information.